docs/secondary-network.md

```diff
@@ -68,10 +68,14 @@ data:
 ```
 
 At the moment, Antrea supports only a single OVS bridge for secondary networks,
-and supports up to eight physical interfaces on the bridge. The physical
-interfaces cannot be the Node's management interface, otherwise the Node's
-management network connectivity can be broken after `antrea-agent` creates the
-OVS bridge and moves the management interface to the bridge.
+and supports up to eight physical interfaces on the bridge.
+
+Note: when you set the Node's primary NIC as the secondary bridge physical interface,
```

Comment:
the secondary bridge physical interface -> a secondary bridge physical interface

Reply:
Updated.